R/normdm.R

```diff
@@ -0,0 +1,51 @@
+#' normdm
+#'
+#' @param k number of groups to be simulated
+#' @param names names for each group
+#' @param n sample size of each group
+#' @param mean mean of simulated data for each group
+#' @param sd standard deviation of simulated data for each group
+#'
+#' @return
+#' @export
+#'
+#' @examples normdm(k=5, names= c("control", "low", "medium", "high", "positive"),n= c(50,50,50,50,40), mean = c(10, 5, 6, 7,8), sd= c(1,5,5,5,5))
+#'
+#'
+normdm <- function(k, names= c(...), n= c(...), mean= c(...), sd= c(...)){
+
+  if(k != (length(n)+length(mean)+length(sd)+length(names))/4){
+    print("k value must match length of names, n, mean, and sd")
+    stop()
+  }
+
+  parameters <<- list(names=names
```

Comment:
move this towards the end, since it is just an output item...just for style's sake